src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -648,7 +648,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         ACTION_ARG(Model::SplitDirection, SplitDirection, SplitDirection::Automatic);
         WINRT_PROPERTY(Model::INewContentArgs, ContentArgs, nullptr);
         ACTION_ARG(SplitType, SplitMode, SplitType::Manual);
-        ACTION_ARG(double, SplitSize, .5);
+        ACTION_ARG(float, SplitSize, .5);

```

Comment:
```suggestion
        ACTION_ARG(float, SplitSize, .5f);
```

Reply:
(I checked and it seems it was the only one. Thanks!)